src/gui/properties/speedplotview.cpp

```diff
@@ -169,6 +170,63 @@ int SpeedPlotView::maxYValue()
     return maxYValue;
 }
 
+namespace
+{
+    //   table of supposed nice steps for grid marks to get nice looking quarters of scale
+    static double roundingTable[] = {1.2, 1.6, 2, 2.4, 2.8, 3.2, 4, 6, 8, 10};
```

Comment:
I'm not sure I understood the code below completely, but aren't powers of 2 would make the nicest scale since we use unit multiplicator value of 1024?

Reply:
No, power of 2 does not fit good because if gives "round values" with many zeros in binary form, not in radix(10) human readable form. And labels having non-zero fractional part makes things even more complicated.

Reply:
Pardon me, I don't understand that. What fractional parts? How they could appear if we use unit prefixes? Are you talking about scale for speeds up to a few bytes per second (when no prefixes are used)?

Reply:
I am talking about case when speed is in range 1kb/s- 4 kb/s or 1mb/s- 4 mb/s etc. Here fractional part in intermediate labels is inevitable.

Reply:
If we just step down to lower unit (mbit/s -> kbit/s for example) there will be to many non-zero digits in labels and no enhancement in appearance.

Reply:
Why not const?